packages/patrol_gen/pubspec.yaml

```diff
@@ -10,11 +10,11 @@ environment:
   sdk: '>=3.5.0 <4.0.0'
```

Comment:
Imo we can bump it to `>=3.8.0 <4.0.0` (and go higher with the dependencies as well). Patrol is already migrated to sdk >3.8.0